slack/pom.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.togglz</groupId>
+    <artifactId>togglz-project</artifactId>
+    <version>2.4.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>togglz-slack</artifactId>
+  <name>Togglz - Slack integration</name>
+  <description>Togglz - Slack integration</description>
+
+  <dependencies>
+
+    <dependency>
+      <groupId>org.togglz</groupId>
+      <artifactId>togglz-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>17.0</version>
```

Comment:
I know that using utility libraries like Guava is usually more convenient. But for library code it is very important do use as few dependencies as possible. Every dependency introduces potential version conflicts. And this is especially true for Guava, which removes deprecated methods/classes after a few releases. So if it is possible to adjust your code to rely on plain JDK classes, you should do so. If it is not possible for certain reasons, we could think about repackaging or just go with the Guava dependency ignoring potential problems in the future.